lib/src/Scene/Qt/QtItemTools.cpp

```diff
@@ -41,11 +43,28 @@ QQuickItem* RepeaterChildWithName(QQuickItem* repeater, const QString& name)
     return item;
 }
 
+QString getObjectId(QObject* object)
```

Comment:
Minor thing, but please change this function name to be capitalized, so that it is more in line with the other freestanding functions. (just for consistency's sake)
Might also make sense to rename it to "GetObject**Name**", as the id is referred to as "Name" by Qt as well (i.e. you get it from the `nameForObject` call).

```suggestion
QString GetObjectName(QObject* object)
```